guides/common/modules/con_smartproxy_performance_tests.adoc

```diff
@@ -7,8 +7,8 @@ We have measured multiple test cases on multiple {SmartProxy} configurations:
 |===
 |{SmartProxy} HW configuration |CPUs |RAM
 |minimal |4 |12 GiB
-|large |8 |24 GiB
-|extra large |16 |46 GiB
+|large |8 |24 GiB
+|extra large |16 |46 GiB
```

Comment:
Is this change removing tabs in favor of spaces? Github UI doesn't tell.

Reply:
Yep, exactly, that is it

Reply:
Or maybe it was not Tab, but some other non-space whitespace char